index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Asad Shareef-Portfolio</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+
+  <header>
+    <nav class='navigation'>
+      <a href="index.html" class="title-top">Welcome</a>
+      <div class="nav-button">
+        <img src="assets/Union.png" />
+      </div>
+    </nav>
+  </header>
+
+
+  <section class="content">
+    <div class="title-body">
+      <h1>Hey there. I'm Asad</h1>
+      <h2 class='highlited-title'>I'm a software developer</h2>
+    </div>
+
+    <p class='about-me-brief'>
+      I can help you build a product , feature or website Look through some of my work and experience! If you like
+      what
+      you see and have a project you need coded,
+      don’t hestiate to contact me.
+    </p>
+
+    <div class='social-icons'>
+      <ul>
+        <li>
+          <a target="_blank" href="#"><img src="assets/twitter.png"
+              alt="Twitter" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/Linkedin icon.png"
+              alt="LinkedIn" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/github.png" alt="GitHub" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/medium.png"
+              alt="GitHub" /></a>
+        </li>
+        <li>
+          <a target="_blank" href="#"><img src="assets/angellist.png"
+              alt="GitHub" /></a>
+        </li>
+      </ul>
+    </div>
+
+    <div class='button-down'>
+      <a target="_blank" href="#"><img src="assets/disabled.png" alt="down" /></a>
+    </div>
+  </section>
+
+  <section class="project">
+    <div class='recent-works'>
+      <h1 class="my-recent-works">My Recent Works</h1>
```

Comment:
- [ ] Nice design so far but let's make look more professional, I noticed that your `h1` " My Recent works" is on the border-radius but if we compare your design to the Figma design   they not really looking the same, kindly remove that in that border and set it as the Figma design :wink:  for your project to look more attractive so that recruiters can reach you out through your work :wink: 

Your Design 
![Screenshot from 2023-02-21 18-54-51](https://user-images.githubusercontent.com/88894525/220411311-531b1593-5afc-4d39-b3f5-86337f380887.png)

Figma Design 
![figmades](https://user-images.githubusercontent.com/88894525/220411401-427d2627-c2c6-4209-a50c-62dcb89e471f.png)

_Note: PLease don't forget to put also that line bar under your `h1`_

Reply:
[@Alfredbis29](https://github.com/Alfredbis29) 
Hello Sir,

I have removed the border from My Recent works and set it as the Figma design.

Reply:
I also gave a line bar under my h1.